elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/framework/SQLUnitTest.java

```diff
@@ -298,14 +299,19 @@ public Query getQuery() {
 
     protected Pattern repeatedWhitespacePattern = Pattern.compile("\\s\\s*");
 
-    public static void init(SQLDialect sqlDialect) {
+    public static void init(String sqlDialect) {
         emf = Persistence.createEntityManagerFactory("aggregationStore");
         EntityManager em = emf.createEntityManager();
         em.getTransaction().begin();
         em.createNativeQuery("DROP ALL OBJECTS;").executeUpdate();
         em.createNativeQuery("RUNSCRIPT FROM 'classpath:create_tables.sql'").executeUpdate();
         em.getTransaction().commit();
 
+        HikariConfig config = new HikariConfig();
```

Comment:
It would be nice if we removed all the entity manager factory code from Aggregation Store and its tests.  We should just use JDBC directly.

Reply:
using org.apache.ibatis.jdbc.ScriptRunner for executing SQL.
For creating datasource, still using emf and persistence.xml. Should I create a new properties file for creating datasource ?

Reply:
Can we use an HJSON config for that?

Reply:
If persistence.xml is just using H2 database, there is a way to run scripts using H2 script runner - or alternatively you can run a script in the JDBC connection URL.

Reply:
working on HJSON config